src/navigators/__tests__/createDrawerNavigator.test.tsx

```diff
@@ -23,72 +23,5 @@ it('renders successfully', () => {
   const App = createAppContainer(MyDrawerNavigator);
   const rendered = renderer.create(<App />).toJSON();
 
-  expect(rendered).toMatchInlineSnapshot(`
-    <RCTView
-      drawerBackgroundColor="white"
-      drawerPosition="left"
-      drawerType="front"
-      drawerWidth={320}
-      hideStatusBar={false}
-      keyboardDismissMode="on-drag"
-      onDrawerClose={[Function]}
-      onDrawerOpen={[Function]}
-      onDrawerStateChanged={[Function]}
-      onGestureRef={[Function]}
-      overlayColor="black"
-      renderNavigationView={[Function]}
-      statusBarAnimation="slide"
-      useNativeAnimations={true}
-    >
-      <View
-        style={
-          Object {
-            "flex": 1,
-          }
-        }
-      >
-        <View
-          collapsable={false}
-          pointerEvents="auto"
-          removeClippedSubviews={false}
-          style={
-            Array [
-              Object {
-                "flex": 1,
-                "overflow": "hidden",
-              },
-              Array [
-                Object {
-                  "bottom": 0,
-                  "left": 0,
-                  "position": "absolute",
-                  "right": 0,
-                  "top": 0,
-                },
-                Object {
-                  "opacity": 1,
-                },
-              ],
-            ]
-          }
-        >
-          <View
-            style={
-              Object {
-                "flex": 1,
-              }
-            }
-          >
-            <View
-              style={
-                Object {
-                  "flex": 1,
-                }
-              }
-            />
-          </View>
-        </View>
-      </View>
-    </RCTView>
-  `);
+  expect(rendered).toMatchInlineSnapshot(`null`);
```

Comment:
🤔

Reply:
I need to figure out how to mock these stuff properly 😓